src/main/java/rife/bld/publish/PomBuilder.java

```diff
@@ -121,6 +124,18 @@ public String build() {
         return StringUtils.stripBlankLines(t.getContent());
     }
 
+    /**
+     * Generates a POM into the given file.
+     *
+     * @since 1.7.1
+     */
+    public void generateInto(PublishInfo info, DependencyScopes dependencies, File file)
```

Comment:
I would either make this static and create a temporary `PomBuilder` instance for this method, or have `generateInto(File file)` to generate the current PomBuilder instance's output. I don't like that this `generateInto` has the side effect of changing the internal state.

Reply:
Fixed